docs/commands.md

```diff
@@ -1,7 +1,7 @@
 # `fnm`
 
 ```
-fnm 1.32.0
+fnm 1.31.1
```

Comment:
I don’t understand why this is not updated, if you’re rebased from main and ran the script. Maybe something is funky there and I’ll fix it?

Reply:
I think it's updated rn after i run it one more time.